lib/Text/BibTeX.pm

```diff
@@ -26,7 +26,9 @@ require DynaLoader;
 
 our $VERSION='0.74';
 
-@ISA = qw(Exporter DynaLoader);
+#@ISA = qw(Exporter DynaLoader);
+use base qw(Exporter DynaLoader);
```

Comment:
Accordingly with base documentation:
Unless you are using the "fields" pragma, consider this module discouraged in favor of the lighter-weight "parent".

So, can you please change to parent? Also, add it to the dependencies.